cluster/images/conformance/Makefile

```diff
@@ -54,7 +54,7 @@ endif
 	chmod a+rx ${TEMP_DIR}/e2e.test
 	chmod a+rx ${TEMP_DIR}/gorunner
 
-	cd ${TEMP_DIR} && sed -i.back "s|BASEIMAGE|${BASEIMAGE}|g" Dockerfile
+	cd ${TEMP_DIR} && sed -i.back "s|ARCH|${ARCH}|g; s|BASEIMAGE|${BASEIMAGE}|g" Dockerfile
```

Comment:
aside: we should really stop using sed for this, docker has supported build arguments for ages, and buildkit even passes along a platform like automatic ARG when using --platforms